### ISSUE TITLE
[BUG] Doesn't Support Organisation Usernames

### ISSUE THREAD

**Issue by RudraSen2** [vn7n24fzkq/github-profile-summary-cards#89]
**Describe the bug**
My Personal GitHub Usernames are working with this but my Organisation Username is not working.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'https://github-profile-summary-cards.vercel.app/demo.html'
2. Type Username as 'Facebook', or any GitHub organisation username.
3. See error - It's all blank

**Expected behaviour**
It should display stats like other personal GitHub usernames.

**Screenshots**
Personal Account:
![image](https://user-images.githubusercontent.com/91307601/169835360-b67dfbed-fd48-4cbc-86f8-88e627030365.png)

Organisation Account:
![image](https://user-images.githubusercontent.com/91307601/169835475-1553fc90-8588-414f-988d-f9e46ff57d76.png)

**Desktop:**
 - OS: Windows 11 Pro Dev Insider Build 25120.1010
 - Browser Google Chrome Latest Version

**Smartphone:**
Not Applicable

**Additional context**
Nothing.

**Comment by leo-schick** [comment]
Same for me, but profile-summary-for-github.com works with organizations quite fine

**Comment by vn7n24fzkq** [comment]
That's odd.

 I will investigate it on the weekend.

**Comment by vn7n24fzkq** [comment]
Hi, I just found the cause.

In GitHub Graph API, user and organization data use different query.

Organization :
```graphql
{
  organization(login: "facebook") {
    id
  }
}
```

User :
```graphql
{
  user(login: "vn7n24fzkq") {
    id
  }
}
```
You can find it in the API explorer
https://docs.github.com/en/graphql/overview/explorer


We can add the getting organization cards feature if this is useful.

What do you think?

**Comment by leo-schick** [comment]
Yes 🙌 would be great to have this feature. Maybe use a parameter to define if the given name is a user or organization

**Comment by vn7n24fzkq** [comment]
Good! I will add this feature.

**Comment by RudraSen2** [comment]
Any progress?

**Comment by vn7n24fzkq** [comment]
Not start this one yet.

Basically, GitHub doesn't allow username duplicates with the organization name, so I will make an auto switch feature here, if the username API was get an error, then use the organization API.

And some data from GitHub GraphQL API may have differences or even not exist between user and organization, so need to do some investigation here.

**Comment by vn7n24fzkq** [comment]
Thanks for the suggestion, and sorry for the long wait! 🙏 Organization support shipped in a recent release — cards now auto-detect orgs, so you can simply pass an org name as `username`:

`https://github-profile-summary-cards.vercel.app/api/cards/profile-details?username=github`

Closing this as done — feel free to reopen if anything's missing!
